host/include/uhd/usrp/multi_usrp.hpp

```diff
@@ -88,7 +88,7 @@ namespace uhd{ namespace usrp{
  *
  * </pre>
  */
-class multi_usrp : boost::noncopyable{
+class UHD_API multi_usrp : boost::noncopyable{
```

Comment:
This is a declaration, so I think the UHD_API macro is appropriate here.

Reply:
This is what I get when the class is dllimport:
3>c:\lib\uhd\host\lib\usrp\multi_usrp.cpp(35): error C2491: 'uhd::usrp::multi_usrp::ALL_GAINS': definition of dllimport static data member not allowed
3>c:\lib\uhd\host\lib\usrp\multi_usrp.cpp(36): error C2491: 'uhd::usrp::multi_usrp::ALL_LOS': definition of dllimport static data member not allowed

I noticed I made some mistakes by adding some extra files, which I tried to revert but apprently it reverted the whole commit. This is the fist time I try to use github and to be honest I used this project to learn about it...